cargo-pgx/src/command/run.rs

```diff
@@ -46,6 +46,9 @@ pub(crate) struct Run {
     /// Use an existing `pgcli` on the $PATH.
     #[clap(env = "PGX_PGCLI", long)]
     pgcli: bool,
+    /// EXPERIMENTAL: Emit a versioned shared object file (and schema)
+    #[clap(long)]
+    versioned_so: bool,
```

Comment:
Do we need this for `cargo pgx run`?  I'm leaning towards no, but could be convinced otherwise...

Reply:
If we do do this for `run`, I suppose we'd want to do it for `test` too?

Reply:
It's not necessary for `run`, but it sure made testing this change a lot easier for me, as I could just `cargo pgx run` instead of having to `install` and then startup and connect to PG. I can add it for test as well. I don't really see the utility in that, but it would make the CLI API surface a little more consistent. What would you prefer?

Reply:
In the latest iteration I've implemented this for `test`.